### DISCUSSION TITLE
How to change the namns displayed in the card?

### DISCUSSION THREAD

**Discussion by AndersNilsson19** [slipx06/sunsynk-power-flow-card#527] [Q&A]
Can i change the word "Daily grid sell" to something other? How? Thanks

**Comment by slipx06** [comment]
Yes under the Grid options

![image](https://github.com/user-attachments/assets/3212b11b-66ed-4122-b6f8-8b4c2a5ba46f)

**Reply by AndersNilsson19** [reply]
![Skärmbild 2024-10-20 152741](https://github.com/user-attachments/assets/82355896-65d1-4cea-a1be-fcf820ec2a83)
Have tried every place to write but thisone dosent change it says "Daglig Belastni.." 2 is missing, but if i cant chang it namn, can i move it to the left some way to get the hole line visibly? For that is my language and the only word that was in my language from start :D

**Comment by slipx06** [comment]
Hi. That's the "Daily Load" which is not possible to change. The Swedish language translation is `"daily_load": "DAGLIG BELASTNING". Is there  a better translation or a way to abbreviate?

**Reply by AndersNilsson19** [reply]
Correct word for that in swedish yes, but the hole word dosent write out, it cut of in the end? Possibly to change it possition to the left ?
Is it not possibly to make it "change possibly" some how in the code? :)  Are u on discord? :) How some good idee for this kind of sun system in HA i want to talk to you about :)

**Reply by AndersNilsson19** [reply]
![Skärmbild 2024-10-23 172046](https://github.com/user-attachments/assets/3c24edba-77ac-448e-9a84-131596044d0b)
Tried to change here but did not work, same text lenght outside the card so it not make it short.

**Reply by slipx06** [reply]
What if I change it to Daglig Bruk?

**Reply by AndersNilsson19** [reply]
"Dagligt bruk"  Tack, men vore också kanon om de går att göra så den faktiskt går att döpa själv till de man vill ha :) 
Mvh

**Reply by slipx06** [reply]
@AndersNilsson19 you can set your own custom label in https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v5.3.0 😄

**Reply by AndersNilsson19** [reply]
> @AndersNilsson19 you can set your own custom label in https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v5.3.0 😄

Thanks man, works well, are u intressted in chatt in discord or wire or something? I have som good idees and pictures and stuff for sun power flow card 2.0 layout pictures and stuff, EVERYONE is going to want to have it if u can make it :D  anders.nilsson  on discord or my wire linc to profile https://account.wire.com/user-profile/?id=ea90f84c-8fcb-4a63-ba75-1ef969f9b7e5

**Reply by slipx06** [reply]
Sure I sent you a request on discord
